src/ui/ToggleEditor.tsx

```diff
@@ -42,7 +43,39 @@ const codeMirrorAPI = ['getValue', 'setValue', 'getRange', 'replaceRange', 'getL
   'indexFromPos', 'focus', 'phrase', 'getInputField', 'getWrapperElement', 
   'getScrollerElement', 'getGutterElement'];
 
-export default @CMBContext class ToggleEditor extends Component {
```

Comment:
The `@Decorator` syntax has been in limbo for a number of years now and I'm not convinced it's every going to be ratified by the powers that be. While both babel and typescript have plugins/options for enabling it as an experimental feature, I've decided to just avoid it in favor of making the code more future proof.

Reply:
I'm _all_ for removing decorators.